distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Table_Helpers.enso

```diff
@@ -486,3 +486,26 @@ get_blank_columns when_any treat_nans_as_blank internal_columns make_column tabl
             0 -> False
             unexpected ->
                 Panic.throw (Illegal_State.Error "Unexpected result: "+unexpected.to_display_text+". Perhaps an implementation bug of Column_Selector.Blank_Columns.")
+
+## PRIVATE
+   Replace a set of columns in the table with a new set of columns. The old
+   columns must all exist in the table, and the list of new columns must be the
+   same length and have the same names.
+replace_columns_with_columns : Table -> Vector Column -> Table
+replace_columns_with_columns table old_columns new_columns =
+    reducer = table-> old_new_pair->
+        old_column = old_new_pair.at 0
+        new_column = old_new_pair.at 1
+        table.set new_column old_column.name
```

Comment:
Let's add set mode for extra safety (it will fail if the column name comes does not exist in the table)
```suggestion
        table.set new_column old_column.name set_mode=Set_Mode.Update
```

Reply:
Done.